addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -111,17 +145,31 @@ def _l10n_it_edi_check_invoice_configuration(self, invoice):
         if not buyer.country_id:
             errors.append(_("%s must have a country.", buyer.display_name))
 
-        # <2.2.1>
-        for invoice_line in invoice.invoice_line_ids:
-            if not invoice_line.display_type and len(invoice_line.tax_ids) != 1:
-                raise UserError(_("You must select one and only one tax by line."))
-
-        for tax_line in invoice.line_ids.filtered(lambda line: line.tax_line_id):
-            if not tax_line.tax_line_id.l10n_it_kind_exoneration and tax_line.tax_line_id.amount == 0:
-                errors.append(_("%s has an amount of 0.0, you must indicate the kind of exoneration.", tax_line.name))
-
         return errors
 
+    def _l10n_it_get_document_type(self, invoice):
+        errors_ordinary = self._l10n_it_edi_check_simplified_invoice_configuration(invoice)
+        errors_simplified = self._l10n_it_edi_check_simplified_invoice_configuration(invoice)
```

Comment:
The two lines above are the same thing...  We should do as little checks as possible to determine if it needs to be simplified or not...  Also country_id == 'IT' is not going to work...  The only check you should do here is whether the address fields are set or not.  (which makes the difference between simplified or not as the other checks have been done already)

Reply:
I was wandering if I could assume that checks are already done. If I couldn't then calling this function could have led to errors